src/backend/renderer/gles2/mod.rs

```diff
@@ -1017,12 +1017,21 @@ impl Drop for Gles2Renderer {
 }
 
 impl Gles2Renderer {
+    /// Get access to the underlying [`EGLContext`].
+    ///
+    /// *Note*: Modifying the context state, might result in rendering issue.
+    /// To make sure a certain modification is safe, check the source.
```

Comment:
"check the source" in a documentation is pretty unusual :laughing: maybe this could be expanded a little?

Reply:
The thing is OpenGL context state is huge and complicated and we *might* adjust the renderer in the future.

If we document every state the renderer needs, we effectively turn this into an api, rather then being an implementation detail. And I dont want to break the api for every small renderer change.

So the idea for downstream *should* be more along the lines of: reset everything you've changed in the end *or* look at the source. 😅

Reply:
done